lib/resources/dstu2/soarian_document_reference.yaml

```diff
@@ -0,0 +1,206 @@
+---
```

Comment:
This content isn't loaded into the document reference page via any definition_table call.  This information would be used by someone looking to post documents, so I think it can be removed

See https://github.com/cerner/fhir.cerner.com/blame/master/content/millennium/dstu2/infrastructure/document-reference.md#L35